src/style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
you need different font-weight for different blocks, use it when you need, not for all html